### DISCUSSION TITLE
Implementation of rate limiter and middleware's

### DISCUSSION THREAD

**Discussion by coderero** [gofr-dev/gofr#548] [Q&A] [answered]
I was wondering that if I can use middleware in GoFr as it is one of my main concerns while using it in production. Is there any way to use middleware in this framework it would be nice and easy to work with. If this is not possible then why it is not possible and what is the actual use case of this framework.

**Comment by aryanmehrotra** [comment, marked as answer]
Hi @coderero, we already have it in pipeline to support custom middlewares, and we are tracking it [here](https://github.com/gofr-dev/gofr/issues/493)
